pkg/kubeapiserver/options/authentication.go

```diff
@@ -236,15 +237,22 @@ func (s *BuiltInAuthenticationOptions) AddFlags(fs *pflag.FlagSet) {
 	if s.ServiceAccounts != nil {
 		fs.StringArrayVar(&s.ServiceAccounts.KeyFiles, "service-account-key-file", s.ServiceAccounts.KeyFiles, ""+
 			"File containing PEM-encoded x509 RSA or ECDSA private or public keys, used to verify "+
-			"ServiceAccount tokens. If unspecified, --tls-private-key-file is used. "+
-			"The specified file can contain multiple keys, and the flag can be specified multiple times with different files.")
+			"ServiceAccount tokens. The specified file can contain multiple keys, and the flag can "+
+			"be specified multiple times with different files. DEPRECATED: If unspecified, "+
+			"--tls-private-key-file is used. This defaulting will no longer occur when the "+
```

Comment:
Simplify to "must be specified when --service-account-signing-key is provided"

Reply:
Done.